### ISSUE TITLE
Overleaf reducing compile timeout to 20 seconds

### ISSUE THREAD

**Issue by hochleitner** [Digital-Media/HagenbergThesis#163]
Overleaf has announced changes, that it will reduce the compile timeout to 20 seconds on the free plan: https://www.overleaf.com/blog/changes-to-free-compile-timeouts-and-servers

This will most likely not be enough to compile our tutorial document but it might also fail for a larger thesis.

I don't think that there is an obvious solution here, except going for a pro account or switching to a local setup.

We should however conduct some tests and add this circumstance to our README to make people aware that they could run into problems.

**Comment by imagingbook** [comment]
Yes, only tests will show. Hopefully the time reduction is compensated by the fact that "free users’ compiles are moving to faster servers" ...

**Comment by hochleitner** [comment]
Since it will still take a semester or more until we get reliable feedback from students and advisors if this causes any noticeable impact, I will add a short paragraph about Overleaf and the possible implications of the restrictions of the free version in Appendix A and the README.md in this repository.

**Comment by imagingbook** [comment]
Just saw the hints in https://t.ly/uvYt5 and got myself a free Overleaf PRO subscription through my IEEE Collabratec account ;-)
Since an IEEE student membership costs very little, this may be an option...

**Comment by hochleitner** [comment]
I opened the branch `content-update` and added a section to appendix A in commits 506a6df1936b7c555679b8592e142ac17c3b1677 (DE) and 8f72ef88678dabec4947e768af27edc8b4113fbe (EN). I also removed TeXmaker as recommended software since its website hasn't been working properly recently. I added Visual Studio Code with the LaTeX Workshop Extension instead since this is favored by many people. I'll create a PR with more content updates soon.

**Comment by imagingbook** [comment]
**IntelliJ** with the **TeXify IDEA** plugin might be another option ...

**Comment by hochleitner** [comment]
> **IntelliJ** with the **TeXify IDEA** plugin might be another option ...

Great, I'll add that, too. I think I'll also remove TeXnicCenter. The latest public release was over ten years ago. They did add a 2.03 alpha build some time last year, but it still seems like active development has been pretty much abandoned.

**Comment by wolfoo2931** [comment]
LaTeX workspaces just dropped in MonsterWriter: https://youtube.com/watch?v=feWZByHoViw  

- Still young, improving rapidly. 
- Free version includes 50MB total storage, no compile limits
- Pro: ~16$/year - 50% off during beta (8$)
